scripts/config/templates/nginx.conf.erb

```diff
@@ -218,20 +218,32 @@ http {
     # 1. There is not a neighbor rendertron header
     # 2. The $request_uri has some kind of /:city--:state-:suffix (no numbers) match
     set $static_render_condition "${http_x_neighbor_rendertron}---${uri}";
+    set $dlp_ab_test_condition "${http_x_neighbor_rendertron}---${is_dlp_v2}---${uri}";
     proxy_intercept_errors on;
 
     # this proxies the request to the prerendered dlp bucket folder if it is not a rendertron request and the URL is DLP-like
-    if ($static_render_condition ~ ^---\/[A-Z][a-zA-Z\-]*--[A-Z][a-zA-Z\-\/]*$) {
+    if ($dlp_ab_test_condition ~ ^---false---\/[A-Z][a-zA-Z\-]*--[A-Z][a-zA-Z\-\/]*$) {
       proxy_pass https://s3.amazonaws.com/neighbor-build-${environment}/prerendered$request_uri;
       break;
     }
 
-    # this proxies the request to the v2 prerendered dlp bucket folder if it is not a rendertron request and the URL is DLP-like
-    if ($static_render_condition ~ "(*UTF8)^---\/locations\/[A-z\-\p{L}'%\/]*$") {
+    if ($dlp_ab_test_condition ~ ^---true---\/[A-Z][a-zA-Z\-]*--[A-Z][a-zA-Z\-\/]*$) {
+      proxy_pass https://s3.amazonaws.com/neighbor-build-${environment}/prerendered$request_uri/v2;
+      break;
+    }
+
+    # this proxies the request to the v1 prerendered dlp bucket folder if it is not a rendertron request and the URL is DLP-like
+    if ($dlp_ab_test_condition ~ "(*UTF8)^---false---\/locations\/[A-z\-\p{L}'%\/]*$") {
       proxy_pass https://s3.amazonaws.com/neighbor-build-${environment}/prerendered$request_uri;
       break;
     }
 
+    # this proxies the request to the v2 prerendered dlp bucket folder if it is not a rendertron request and the URL is DLP-like
+    if ($dlp_ab_test_condition ~ "(*UTF8)^---true---\/locations\/[A-z\-\p{L}'%\/]*$") {
```

Comment:
do we care about keeping the ab test if someone is hitting an old url structure?

Reply:
I don't think it hurts. It'll still fire the same events

Reply:
I guess I'm more concerned about the complexity of making everything backwards compatible over time. at some point we should just deprecate support for the old url and maybe that is now